ChangeLog.d/systematically_store_bit_size_3740.txt

```diff
@@ -0,0 +1,6 @@
+Changes
+   * Always store key bits attribute in PSA key storage. Previously stored
+     keys should be augmented with the bits value to be compatible with the
+     updated version. Also removes the stored bits field from the storage
+     format for the dynamic Secure Element driver. Previously stored keys
+     must be updated to remove the stored bits field. #3740
```

Comment:
This is both too detailed and not detailed enough. The changelog file is intended for library users. The short version is that we don't provide an upgrade path. To upgrade an existing device, you need to either wipe out the storage, or work out how to upgrade existing key objects from the format description, you can't work out what to do just from the changelog entry.

Reply:
I have attempted a re-write now that gives the same general guideline as in your comment, and a pointer to the format documentation.